raiutils/raiutils/models/models.py

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) Microsoft Corporation
```

Comment:
what do you think about calling this file model_utils?  I always found it confusing to have the folder and file be the same name, eg models/models.  We could even break it into constants.py (or model_constants.py?) vs model_utils.py eventually, but probably it's too much to break it at this point, since there isn't a lot of code here yet.

Reply:
The suggested change was made under https://github.com/microsoft/responsible-ai-toolbox/pull/1295.